### PR TITLE
feat(sbb-header, sbb-footer): introduce wide / full width variant 

### LINKED ISSUE #1500: sbb-header: provide css vars to remove spacing
### Request type

Request for a new component

### Header variant
For fullscreen layouts a header variant is needed, where the offset is replaced by a responsive spacing size XS. This header has a fixed position, so there is no sticky or scroll-up behaviour.

### Is there a design proposal?

Yes

### Link to design proposal file (optional)

https://www.figma.com/file/KCNPyUuLqJkymyVSP1EIaY/05%3A-Design-%E2%80%93-Atlas-%F0%9F%9A%A7?node-id=6630%3A351297&t=eAsGvuf0SLjIqZHP-1

### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #1500

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
However, I wonder whether we should provide a global CSS class or a property to enable this mode instead of having the consumer manage CSS variables. Thoughts?

**Comment by jeripeierSBB** [comment]
> LGTM 👍 However, I wonder whether we should provide a global CSS class or a property to enable this mode instead of having the consumer manage CSS variables. Thoughts?

Let us discuss this, it was the reason I did not merge it myself ;-)

File: src/components/sbb-footer/sbb-footer.tsx [hunk 1]
```diff
@@ -18,6 +18,9 @@ export class SbbFooter {
   /** Negative coloring variant flag. */
   @Prop({ reflect: true }) public negative = false;
 
+  /** Whether the footer should be stretched to full width. */
```
Comment (kyubisation) [inline src/components/sbb-footer/sbb-footer.tsx:21]:
```suggestion
  /**
   * Whether to allow the footer content to stretch to full width.
   * By default, the content has the appropriate page size.
   */
```

File: src/components/sbb-header/sbb-header.tsx [hunk 2]
```diff
@@ -14,6 +14,9 @@ export class SbbHeader {
   /** Used to display a box-shadow below the component on y-axis scroll whether set to true. */
   @Prop({ reflect: true }) public shadow = false;
 
+  /** Whether the header should be stretched to full width. */
```
Comment (kyubisation) [inline src/components/sbb-header/sbb-header.tsx:17]:
```suggestion
  /**
   * Whether to allow the header content to stretch to full width.
   * By default, the content has the appropriate page size.
   */
```

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Two comments about documentation.

**Pull request merged**